plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/runtime/policy/PolicyDataProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.runtime.policy;
+
+/**
+ * Interface for data policy operations
+ */
+public interface PolicyDataProvider {
+
+    /**
+     * Retrieve policy value from system environment 
+     *
+     * @param propertyName - data policy
+     * @return - boolean policy value
+     */
+    boolean getDataPolicyFromSystem(String propertyName);
```

Comment:
We don't need these function in interface. Leave just isDataPolicyEnabled.

Reply:
Done